eq-author-api/middleware/identification/index.test.js

```diff
@@ -10,10 +10,25 @@ const keysJson = JSON.parse(JSON.stringify(keysYaml));
 
 const { createUser } = require("../../utils/datastore");
 
-describe("auth middleware", () => {
+const invalidJwtToken = mockJwt.sign({ id: "invalid.token" }, uuid.v4());
+
+jest.mock("./verifyJwtToken", () => {
+  return {
+    verifyJwtToken: accessToken => {
+      const jwtToken = mockJwt.decode(accessToken);
+      return jwtToken.id === "invalid.token"
+        ? {
+            error: true,
+          }
+        : mockJwt.decode(accessToken);
+    },
+  };
+});
+
+describe("auth middleware", async () => {
   let logger;
 
-  beforeEach(() => {
+  beforeEach(async () => {
```

Comment:
Does beforeEeach need to be async? Also on line 52.

Reply:
nope. I must have had an await call in there at some point. I've removed the async on both lines.